rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -2430,6 +2447,21 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> gui_settings, std
 	}
 }
 
+void settings_dialog::refresh_countrybox()
+{
+	const auto& vec_countries = countries::g_countries;
+	const std::string cur_country = m_emu_settings->GetSetting(emu_settings_type::PSNCountry);
+
+	ui->psnCountryBox->clear();
+
+	for (const auto& [cnty, code] : vec_countries)
+	{
+		ui->psnCountryBox->addItem(QString::fromUtf8(cnty.data(), static_cast<int>(cnty.size())), QString::fromUtf8(code.data(), static_cast<int>(code.size())));
```

Comment:
I think you can just use the const char* constructor `QString(const char *str)`
```suggestion
		ui->psnCountryBox->addItem(QString(cnty.data()), QString(code.data()));
```

Reply:
ah. it does the same under the hood. nevermind then